themes/developers/dev/assets/sass/plugins/_give.scss

```diff
@@ -0,0 +1,108 @@
+/* ## Give plugin (see https://givewp.com/)
+---------------------------------------------------*/
+.give-form-wrap .give-form-title {
+	text-align: center;
+	padding: 20px 0 20px;
+	@include rem(font-size, $give__Donation_Form_Title_Desktop--font-size);
+	font-weight: 700;
+	font-style: italic;
+
+	@media only screen and (max-width: $breakpoint__screen-size--small) {
+		@include rem(font-size, $give__Donation_Form_Title_Mobile--font-size);
+	}
+}
+
+// Flex container for the form's featured image & caption.
+.donation-form-featured-image {
+	display: flex;
+	flex-direction: column;
+	flex-wrap: nowrap;
+	justify-content: space-around;
+	align-items: center;
+
+	// Featured image on donation form
+	.featured-image  {
+		order: 1;
+		flex-grow: 1;
+		margin-bottom: 20px;
+	}
+
+	// Image caption on donation form.
+	.post-1411-caption {
+		padding: 10px 0 10px;
+		order: 2;
+		flex-grow: 1;
+		@include rem(font-size, $give__Donation_Form_Featured_Image_Caption_Desktop--font-size);
+
+		@media only screen and (max-width: $breakpoint__screen-size--small) {
+			@include rem(font-size, $give__Donation_Form_Featured_Image_Caption_Mobile--font-size);
+			text-align: center;
+			line-height: $featured-image-caption_mobile--line-height;
+		}
+	}
+}
+
+#give-form-content-1349 > blockquote:nth-child(1) {
```

Comment:
@rgadon107 The form's ID will be different when you move it to the production server. You need a different way to target this element.